BuzzFeed/dist/presence.js

```diff
@@ -68,17 +68,17 @@ presence.on("UpdateData", () => __awaiter(this, void 0, void 0, function* () {
         data.details = "Reads a Post:"
         data.state = `${posttitle.textContent.trim()}`;
     } else if (pages[page] || pages[page.slice(0, -1)]) {
-        data.details = "Shows to a Page:"
+        data.details = "looks to a Page:"
         data.state = pages[page] || pages[page.slice(0, -1)];
     } else if (page.includes("/search")) {
         data.details = "Searching:"
         data.state = document.title;
         data.smallImageKey = "logo";
     } else if (user && user.textContent != "") {
-        data.details = "Shows to User Profile:"
+        data.details = "looks to a user Profile:"
         data.state = user.textContent.trim()
     } else {
-        data.details = "Shows to a Page:"
+        data.details = "looks to a Page:"
```

Comment:
Dont you mean:
"Viewing page:"

Reply:
Aaa yes Sorry My english is a2. Thats Why my English is bad